projectile.el

```diff
@@ -3746,7 +3746,7 @@ Replace STRING in DIR-PATH with REPLACEMENT."
   (let* ((project-root (projectile-project-root))
          (relative-dir (file-name-directory (file-relative-name dir-path project-root))))
     (projectile-expand-root
-     (replace-regexp-in-string string replacement relative-dir))))
+     (replace-regexp-in-string string replacement relative-dir t))))
```

Comment:
We should also leave some TODO to use `(string-replace FROMSTRING TOSTRING INSTRING)` (introduced in Emacs 28) once we start targeting Emacs 28 in the distant future.

Reply:
I've updated the PR to include a TODO comment.